snapcraft/internal/repo/_deb.py

```diff
@@ -300,6 +300,34 @@ def refresh_build_packages(cls) -> None:
                 "failed to run apt update"
             ) from call_error
 
+    @classmethod
+    def _check_if_all_packages_installed(cls, package_names: List[str]) -> bool:
+        # If any requested package is not installed, return False. Will check
```

Comment:
With minor edits, this could be a docstring for the method.

Reply:
Certainly could be, and I'll change it!  Since it's private I didn't think it necessary.

FWIW, pep8 says `Write docstrings for all public modules, functions, classes, and methods. Docstrings are not necessary for non-public methods, but you should have a comment that describes what the method does. This comment should appear after the def line.`

Would you prefer to docstring all methods, at least lightly?

Reply:
documenting docstrings instead of code comments is nice when using the method elsewhere and the IDE supports pop-over help

Reply:
I don't mind it staying private, as Repo is exported to plugin users and used in v1

Reply:
Understood, thanks!

Reply:
Done BTW